_docker/control.rb

```diff
@@ -357,9 +357,6 @@ def initialise_environment(environment, pull_drupal_data_image, system_exec)
   environment = load_environment(tasks)
   initialise_environment(environment, tasks[:docker_pull], system_exec)
 
-  #the docker url is taken from DOCKER_HOST env variable otherwise
-  Docker.url = tasks[:docker] if tasks[:docker]
-
```

Comment:
@LightGuard I still use docker-machine and I have to resort to running `eval $(docker-machine env redhat)` to correctly interact with the `Docker` daemon. I additionally do that before running the `control.rb` script, as if you don't it fails with:

```
- Executing 'docker' with command 'build' and args ["--tag=developer.redhat.com/base:2.0.0", "./base"]
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
Error running docker
```

I'm happy to drop the `-d` option as I think very few people are using it and the "work-around" is to run `eval $(docker-machine env <machine_name>)` before using control.rb, which you would need to do anyways to interact with the Docker containers in any meaningful way.